great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -804,7 +880,7 @@ def _create_batch_spec(self, batch_spec_kwargs: dict) -> BatchSpec:
         """
         raise NotImplementedError
 
-    def as_selectable(self) -> sqlalchemy.Selectable:
+    def as_selectable(self) -> sa.Selectable:
```

Comment:
We had imported `sqlalchemy` as `sqlalchemy` and as `sa` so all these changes are cleanup.